tls/s2n_tls13_key_schedule.c

```diff
@@ -36,13 +36,7 @@ static S2N_RESULT s2n_zero_sequence_number(struct s2n_connection *conn, s2n_mode
     RESULT_ENSURE_REF(conn);
     RESULT_ENSURE_REF(conn->secure);
     struct s2n_blob sequence_number = { 0 };
```

Comment:
How did we end up with two static s2n_zero_sequence_number functions?

Reply:
lol I didn't even notice that 🤦‍♀️ But I guess by not making this change earlier...